Rx/v2/src/rxcpp/rx-operators.hpp

```diff
@@ -97,8 +97,6 @@ class operator_factory
 
 #include "operators/rx-connect_forever.hpp"
 #include "operators/rx-lift.hpp"
-#include "operators/rx-multicast.hpp"
-#include "operators/rx-publish.hpp"
```

Comment:
The include structure should be restored, please take a look at the last commit.